doc/tutorials/ios-app.md

```diff
@@ -391,8 +391,18 @@ rules.
 
 ### Install the app on a device
 
-The easiest way to install the app on the device is to launch Xcode and use the
-`Windows > Devices` command. Select your plugged-in device from the list on the
+You can install the app on a physical device using the bazel run command, just as you would for a simulator:
+
+```bash
+bazel run //:iOSApp --ios_multi_cpus=arm64
+```
```

Comment:
We should also recommend users use a `common --flag_alias=ios_device=@rules_apple//apple/build_settings:ios_device` in their `.bazelrc` so they can just do `bazel run //FooApp --ios_device="id"`

Reply:
Done